pkg/cmd/server/api/types.go

```diff
@@ -92,6 +92,10 @@ type MasterConfig struct {
 type ProjectRequestConfig struct {
```

Comment:
I know that it's not part of your change, but shouldn't the type itself also have godoc?